### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by HenryNovoa**

### CODE REVIEW

File: staff/henry-novoa/app-template/contact.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <a href="./contact.html">Contact</a>
+            <a href="login.html">Login</a>
+            <a href="register.html">Register</a>
+    
+    </nav>
+
+    <section>
+        
+         <address>
+             Email:<a href="">lala@lolo</a> <br>
```
Comment (Sergioluz90) [inline staff/henry-novoa/app-template/contact.html]: I think the way to send an email is:  href="mailto:PepitoPepote@gmail.com"

File: staff/henry-novoa/app-template/news.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                <a href="./news.html">News</a>
+                <a href="./about.html">About</a>
+                <a href="./contact.html">Contact</a>
+                <a href="login.html">Login</a>
+                <a href="register.html">Register</a>
+        
+        </nav>
+
+        <article>
+            <img src="https://imgc.allpostersimages.com/img/print/posters/jagodka-black-labrador-retriever-profile_a-G-12263989-14258384.jpg" alt="Black dog labrador jagodka-black-labrador-retriever-profile_a-G-12263989-14258384" alt="black labrador retriever">
```
Comment (gbaste) [inline staff/henry-novoa/app-template/news.html]: you only need one alt in each image.

File: staff/henry-novoa/app-template/contact.html (hunk 1 above)
Comment (HenryNovoa) [reply staff/henry-novoa/app-template/contact.html]: Well noted, I'm on it

File: staff/henry-novoa/app-template/news.html (hunk 2 above)
Comment (HenryNovoa) [reply staff/henry-novoa/app-template/news.html]: Well noted, I'm on it, small typo from copying.

File: staff/henry-novoa/app-template/contact.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <header>
+        <h1>Contact</h1>                 
+    </header>
+    <nav>
```
Comment (seand52) [inline staff/henry-novoa/app-template/contact.html:13]: It's better to have each item in a list in your nav bar. so you would have  an "ul" and each anchor tag would be inside a "li" tag

Comment (manuelbarzi) [reply staff/henry-novoa/app-template/contact.html:13]: agree

Comment (HenryNovoa) [reply staff/henry-novoa/app-template/contact.html:13]: I'm on it

**Pull request merged**